test/jdk/java/security/SignedJar/spi-calendar-provider/provider/baz/CalendarDataProviderImpl.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) 2012, 2022, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Did you copy this code from somewhere else? In any case, the impl seems trivial so I don't think it needs to have the Oracle copyright. Otherwise, you should probably use your own company here, and just put 2022 as the date.

Reply:
OK.

Reply:
Fixed now.